### PR TITLE
minor string fixes

### PR DESCRIPTION
**Pull request by un1versal**
Takes care of some minors and brings it inline with Kodi strings and the other PR's for these adsp addons

### CODE REVIEW

File: adsp.freesurround/resources/language/English/strings.po [hunk 1]
```diff
@@ -22,15 +22,15 @@ msgid "Free Surround"
 msgstr ""
 
 msgctxt "#30001"
-msgid "Free Surround ..."
+msgid "Free Surround..."
 msgstr ""
 
 msgctxt "#30002"
-msgid "FreeSurround, a new surround processor. Is based on foo_dsp_fsurround from pro_optimizer on foobar2000 0.9.x."
+msgid "FreeSurround, is a new surround processor. Is based on foo_dsp_fsurround found on pro_optimizer in foobar2000 0.9.x."
```
Comment (MartijnKaijser) [inline adsp.freesurround/resources/language/English/strings.po]: .... It is based....  or .....It's based....

File: adsp.freesurround/resources/language/English/strings.po [hunk 2]
```diff
@@ -22,15 +22,15 @@ msgid "Free Surround"
...
+msgid "FreeSurround is meant to be the Kodi equivalent of your hi-fi receiver's Dolby ProLogic II button (except that it uses a different algorithm).\n\nIts purpose is to decode surround information from your stereo music including mp3, and to produce multichannel output from it. It works with any stereo music containing surround info in a coding scheme compatible with any the following decoders:\n - Dolby Surround\n - Dolby ProLogic\n - Dolby ProLogic II\n - SRS CircleSurround\n - SRS CircleSurround II\n - DTS Neo:6\n - Lexicon Logic7\n - Stereo Quadrophonic (SQ)\n - Quadrophonic Stereo (QS)\n\nIs based on foo_dsp_fsurround from pro_optimizer in foobar2000 0.9.x"
```
Comment (MartijnKaijser) [inline adsp.freesurround/resources/language/English/strings.po]: has several double \n\n  Are they needed?

Comment (un1versal) [inline adsp.freesurround/resources/language/English/strings.po]: No idea this crashes kodi for me so I cant tell what's needed only corrected the obvious.

Edit Ive removed one \n anyway.

File: adsp.freesurround/resources/language/English/strings.po (hunk 1 above)
Comment (un1versal) [inline adsp.freesurround/resources/language/English/strings.po]: Well spoted, and done.

File: adsp.freesurround/resources/language/English/strings.po [hunk 3]
```diff
@@ -22,15 +22,15 @@ msgid "Free Surround"
 msgstr ""
 
 msgctxt "#30001"
-msgid "Free Surround ..."
+msgid "Free Surround..."
 msgstr ""
 
 msgctxt "#30002"
-msgid "FreeSurround, a new surround processor. Is based on foo_dsp_fsurround from pro_optimizer on foobar2000 0.9.x."
+msgid "FreeSurround, is a new surround processor. It's based on foo_dsp_fsurround found on pro_optimizer in foobar2000 0.9.x."
```
Comment (MartijnKaijser) [inline adsp.freesurround/resources/language/English/strings.po]: Second sentence is still wrong somehow.
"It's based on foo_dsp_fsurround as found in pro_optimizer from foobar2000 0.9.x."  or something like that

Comment (un1versal) [inline adsp.freesurround/resources/language/English/strings.po]: Should be all better now.

**Pull request merged**